app/controllers/group_health_controller.rb

```diff
@@ -132,7 +145,8 @@ def role_types
   end
 
   def group_types
-    respond(Group.all_types.map do |type|
+    respond(Group.all_types.excluding(Group::InternesGremium,
+                                      Group::InternesAbteilungsGremium).map do |type|
```

Comment:
Könntest du diese Liste von Gruppentypen in einer Konstante festhalten, und dann DRY auch im `EXCLUDE_INTERNES_GREMIUM` verwenden?